### PR TITLE
Update tutorial-setup doc

### PR DESCRIPTION
**Pull request by SaravanaStorageNetwork**
Update the version number as a variable one

### CODE REVIEW

File: modules/ROOT/pages/tutorial-setup.adoc [hunk 1]
```diff
@@ -83,7 +83,7 @@ To make the tutorial simpler, you should rename the image that we have just down
 
 [source,bash]
 ----
-mv fedora-coreos-32.20200715.3.0-qemu.x86_64.qcow2 fedora-coreos.qcow2
+mv fedora-coreos-<version_number>-qemu.x86_64.qcow2 fedora-coreos.qcow2
```
Comment (lucab) [inline modules/ROOT/pages/tutorial-setup.adoc:86]: There are a few more cases like this in the rest of the page. I think it's easier to set a `$RELEASE` variable and use it later on. Something like:
```
coreos-installer download -p qemu -f qcow2.xz --decompress
RELEASE="32.20200715.3.0" # Change this according to the current version that has been downloaded.
```

**Review by lucab — COMMENTED** [review]
Thanks for the PR. I've left a different suggestion in review.

File: modules/ROOT/pages/tutorial-setup.adoc (hunk 1 above)
Comment (bgilbert) [reply modules/ROOT/pages/tutorial-setup.adoc:86]: coreos-installer prints the full path to the downloaded file to stdout, so we could just capture that into a variable.

**Comment by travier** [comment]
I think we can close this one now that we have https://github.com/coreos/fedora-coreos-docs/pull/356. Thanks for the PR!

**Pull request closed without merging**